modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You installed the OpenShift CLI (`oc`).
+* Your oc client must be updated to at least verion 4.13.0. 
+* Your {product-title} cluster must be installed on either the AWS or Azure platform. 
```

Comment:
@Prashanth684 @LalatenduMohanty can you please confirm if we should mention `connected cluster` here?

Reply:
yes, but isn't that implied for upgrades in general?

Reply:
Ah - makes sense. IMO we should cover the most common scenario and then supplement iteratively if necessary.  I'll go with option B for now. @shellyyang1989 @Prashanth684 any concerns?

Reply:
Sounds good to me! Thank you.

Reply:
+1 from me.